src/sass/header.sass

```diff
@@ -141,7 +135,7 @@
                     &:hover
                         color: white
             .menu.lvl3
```

Comment:
maybe it's better to create new menu, because this classes is not supported BEM

Reply:
Should I make another menu without foundation?